docs/concept/buckconfig.soy

```diff
@@ -2862,10 +2862,23 @@ your <code>.buckjavaargs</code> file</a>:
 {call buckconfig.entry}
   {param section: 'ndk' /}
   {param name: 'app_platform' /}
-  {param example_value: 'android-21' /}
   {param description}
+  <p>
     The android platform libraries that the code is targeting.  This is equivalent to the{sp}
     <code>APP_TARGET</code> in the NDK build system.  The default is <code>android-16</code>.
+  </p>
+  <p>
+  This setting can also be set on a per-CPU ABI basis. Each key should begin with the{sp}
+  prefix <code>app_platform-</code>, followed by the CPU ABI name. ABI-specific values{sp}
+  have higher precedence.
+  </p>
+  {/param}
+  {param raw_example}
+{literal}<pre class="prettyprint lang-ini">
+[ndk]
+  app_platform = android-19
+  app_platform-arm64 = android-21
```

Comment:
Can we have a map instead? Buck configuration supports keeping maps in a single option. For example:

```
app_platform_per_abi = arm64 => android-21, armv7 => android-23
```

Reply:
Sure, though I did it this way to try to generally be consistent with https://buckbuild.com/concept/buckconfig.html#halide.target. Still want me to change it?

Reply:
Yeah, could you change that to using a map?